stack-lts-12.26.yaml

```diff
@@ -6,3 +6,6 @@ extra-deps:
   - esqueleto-3.4.0.1@sha256:85077c6b11e06e0ac4a5cb07fb6b71448dcd2dc1e9831ea8115cb9090c342500,5869
   - megaparsec-7.0.4@sha256:a7397151601cbe6b8f831f8bdad1a10118dcd6d9a7ee50d6bbdcfbd1181b4ba2
   - persistent-2.11.0.1@sha256:c33f00adfed435bbf406d70aa6e5471974a4afcc1a8da3a5ef4e1af0ba5f9884,4702
+  - serialise-0.2.6.0
+  - strict-0.5@sha256:76f07389001fa7a9036cbf13a1753c0f16556bde3b79c862a8bbe1201aac2326,4074
+  - these-1.2@sha256:69206d92ed38c45da07a34a86b8fafbefc7cec511ca7003ba18736a61639d03b,2844
```

Comment:
Missing newline at end of file.

As mentioned in Slack, you can feel free to just remove the `@...` checksums. We no longer do that.